src/pages/patientView/clinicalInformation/ClinicalInformationSamplesTable.tsx

```diff
@@ -3,36 +3,44 @@ import { ClinicalDataBySampleId } from "../../../shared/api/api-types-extended";
 import convertSamplesData, {IConvertedSamplesData} from "./lib/convertSamplesData";
 import {SampleLabelHTML} from "../../../shared/components/sampleLabel/SampleLabel";
 import {Table as DataTable, Thead, Th} from "reactable";
+import LazyMobXTable from "shared/components/lazyMobXTable/LazyMobXTable";
 import TableHeaderControls from "shared/components/tableHeaderControls/TableHeaderControls";
+import {ClinicalAttribute} from "../../../shared/api/generated/CBioPortalAPI";
+
 
 interface IClinicalInformationSamplesTableProps {
     samples?: Array<ClinicalDataBySampleId>;
 }
 
+export interface ISampleRow {
+    attribute:string;
+    [key:string]:string|number|ClinicalAttribute;
+}
+
+class SampleTableComponent extends LazyMobXTable<ISampleRow> {
+}
+
 export default class ClinicalInformationSamplesTable extends React.Component<IClinicalInformationSamplesTableProps, void> {
 
     public render() {
         const sampleInvertedData = convertSamplesData(this.props.samples);
         const tableData = this.prepareData(sampleInvertedData);
         const headerCells = this.buildHeaderCells(sampleInvertedData);
-        return (
-            <div>
-                <div>
-                    <TableHeaderControls className="pull-right" tableData={tableData} />
-                </div>
-                <DataTable className="table table-striped table-border-top" data={tableData} >
-                    <Thead>{ headerCells }</Thead>
-                </DataTable>
-            </div>
-        );
+        const columns = headerCells.map((cell) => (
+            {name: cell.props.children,
+             render: (data:ISampleRow)=><span>{data[cell.props.column]}</span>}
+        ));
+        return <SampleTableComponent columns={columns} data={tableData} showPagination={false} showColumnVisibility={false}/>;
     }
 
     public buildHeaderCells(sampleInvertedData: IConvertedSamplesData) {
```

Comment:
Feels like it shouldn't be used. `Th` components are already generated by `LazyMobXTable`, right?